addons/website_sale/views/templates.xml

```diff
@@ -1921,7 +1921,10 @@
             <p class="text-muted">
                 <t t-foreach='website.shop_extra_field_ids' t-as='field' t-if='product[field.name]'>
                     <b><t t-esc='field.label'/>: </b>
-                    <t t-if='field.field_id.ttype != "binary"'>
+                    <t t-if='field.field_id.ttype == "selection"'>
```

Comment:
it will be more readable to change condition if it is not True/False now...

if type == "selection"
elif type ==" binary"
else
   ...

Else fix LGTM

Reply:
naive question, but we have a Selection widget...

`return html_escape(pycompat.to_text(options['selection'][value]) or u'', options)`

did you check why it doesn't work?